src/core/routes.py

```diff
@@ -1,8 +1,8 @@
 import asyncio
+from functools import wraps
 
 from aiohttp.web_urldispatcher import UrlDispatcher
-from .base import BaseView
-from functools import wraps
+from core.base import BaseView
```

Comment:
One small "trolling comment" for @pzdeb ;)
According to Pep 8 importing you should  put a blank line between each group of imports:
- standard library imports
- related third party imports
- local application/library specific imports

In your case it's line number 5.